rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/parametric/ILanguageContributions.java

```diff
@@ -110,9 +110,9 @@ public static interface ScheduledCalculator extends BiFunction<ISourceLocation,
 
     @FunctionalInterface
     // To conveniently pass methods `documentation`, `definitions`,
-    // `references`, and `implementations` as parameters.
+    // `references`, `actions` and `implementations` as parameter.
     public static interface OndemandCalculator {
```

Comment:
can it now just extend `Function<IList, InterruptibleFuture<ISet>`? or maybe replace it? as in it looks like this interface was added since there exist no `TriFunction`?

Reply:
I don't know why this was added. It wasn't me :-) I'll try and clean it up.